go.mod

```diff
@@ -271,8 +271,8 @@ replace (
 	// TODO: remove it: https://github.com/cosmos/cosmos-sdk/issues/13134
 	github.com/dgrijalva/jwt-go => github.com/golang-jwt/jwt/v4 v4.4.2
 	github.com/ethereum/go-ethereum => github.com/crypto-org-chain/go-ethereum v1.10.20-0.20240926023215-d2275b4afb9a
-	// release/v1.4.x
-	github.com/evmos/ethermint => github.com/crypto-org-chain/ethermint v0.6.1-0.20250109075157-62042e2252f6
+	// release/v1.4.x-2
+	github.com/evmos/ethermint => github.com/crypto-org-chain/ethermint v0.6.1-0.20250409022716-434f46d2a46c
```

Comment:
can you tell me which branch you are using and the current workflow?

`develops` vs `release.vxx` , what are the differences?

Reply:
it says [release/v1.4.x-2](https://github.com/crypto-org-chain/ethermint/commits/release/v1.4.x-2) align with https://github.com/crypto-org-chain/cronos/blob/release/v1.4.x/go.mod

Reply:
you can use the commit and go mod tidy, which'll correct the timestamp